src/shared/proto/geometry.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
-message PointMsg
+package TbotsProto;
```

Comment:
nit: Is it common / good practice to suffix `Proto` to proto package declarations? It seems a bit redundant and makes the code-generated names longer. Not syre how much benefit there is to knowing if some generated thing is specifcally proto or not.

Reply:
I want the naming to disambiguate with internal data types, for example the `Angle` geometry class vs the `Angle` protobuf class. There are several options for doing this:
* TbotsProto::Angle
* Tbots::Angle
* Tbots::AngleMsg
While Tbots::Angle is shortest, I thought it was confusing to have a namespace called Tbots (is the rest of our code base not Tbots code?). We might also be able to get away with `package Proto` and use that across SSL protobufs and our own protobufs if there aren't any name conflicts, but that's trickier to pull off.

Reply:
I agree just having `TBots` is too ambiguous. I also prefer including `Proto` near the start of the namespace / package instead of appending `Msg` to the message names. So all in all, I'd be fine with either using google's style guide, or using our own simple version with something like `TBotsProto` so we can remove the `Msg` suffixes.

Reply:
OK I will leave it as `TbotsProto`